Source/Shared/Cache/BasicHybridCache.swift

```diff
@@ -9,6 +9,12 @@
  Cachable protocol. It's two layered cache (with front and back storages)
  */
 public class BasicHybridCache: NSObject {
+  enum CacheError: Error {
```

Comment:
Then I have to use `Swift.Error` in completion closures, and I don't really like it.